docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -60,31 +60,31 @@ behavior:
   determines how an Agent makes decisions. More on Behavior Parameters in
   the next section.
 * **Max Step** — Defines how many simulation steps can occur before the Agent
-  decides it is done. In 3D Balance Ball, an Agent restarts after 5000 steps.
+  episode ends. In 3D Balance Ball, an Agent restarts after 5000 steps.
 
 Perhaps the more interesting aspect of an agents is the Agent subclass
 implementation. When you create an Agent, you must extend the base Agent class.
 The Ball3DAgent subclass defines the following methods:
 
-* agent.AgentReset() — Called when the Agent resets, including at the beginning
+* `Agent.OnEpisodeBegin()` — Called when the Agent resets, including at the beginning
   of a session. The Ball3DAgent class uses the reset function to reset the
   agent cube and ball. The function randomizes the reset values so that the
   training generalizes to more than a specific starting position and agent cube
   attitude.
-* agent.CollectObservations(VectorSensor sensor) — Called every simulation step. Responsible for
+* `Agent.CollectObservations(VectorSensor sensor)` — Called every simulation step. Responsible for
   collecting the Agent's observations of the environment. Since the Behavior
   Parameters of the Agent are set with vector observation
   space with a state size of 8, the `CollectObservations(VectorSensor sensor)` must call
   `VectorSensor.AddObservation()` such that vector size adds up to 8.
-* agent.AgentAction() — Called every simulation step. Receives the action chosen
+* `Agent.OnActionReceived()` — Called every simulation step. Receives the action chosen
```

Comment:
Not directly related to this PR, but "Called every simulation step" is not correct.